From f9f7f1231e835cc335fdc81b30544b8da35f79b3 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Sat, 8 Jul 2023 18:58:44 +0200
Subject: [PATCH] preload/clickpreload.c: define _LARGEFILE64_SOURCE

Defining this will make the type off64_t available which is used in this
file. Otherwise we'll get a compilation error.

clickpreload.c:221:29: error: unknown type name 'off64_t'; did you mean 'off_t'?
  221 | int sync_file_range(int fd, off64_t offset, off64_t nbytes, unsigned int flags)
      |                             ^~~~~~~
      |                             off_t
---
 preload/clickpreload.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/preload/clickpreload.c b/preload/clickpreload.c
index b29088a..733b66c 100644
--- a/preload/clickpreload.c
+++ b/preload/clickpreload.c
@@ -20,6 +20,7 @@
  */
 
 #define _GNU_SOURCE
+#define _LARGEFILE64_SOURCE
 
 #include <assert.h>
 #include <dlfcn.h>
-- 
2.41.0

